contracts/raffles/src/state.rs

```diff
@@ -130,6 +130,7 @@ pub struct RaffleOptions {
     pub max_ticket_per_address: Option<u32>, // max amount of tickets able to bought per address
     pub raffle_preview: u32,               // ?
     pub min_ticket_number: Option<u32>,    // Minimum ticket number for a raffle to close.
+    pub one_winner_per_asset: bool, // Allows to set multiple winners per raffle (one per asset)
```

Comment:
I do wonder if we're due for some refactoring the `RaffleOptions` to attempt to ensure that impossible states are impossible to represent. but without anything particularly concrete to recommend i'm going to approve the changes in the pr.